src/abstractmps.jl

```diff
@@ -1942,11 +1955,20 @@ by site according to the site indices `sites`.
    in `sites` and `leftinds` will be dangling off of the right side of the MPS/MPO.
 - `orthocenter::Integer = length(sites)`: the desired final orthogonality
    center of the output MPS/MPO.
+- `tags::Vector{TagSet} = [defaultlinktags(i) for i in 1:(length(sites) - 1)]`:
+   the tags to use for the link indices. The length of `tags` must be
+   `length(sites) - 1`. The default is to use the default link tags for each
+   site.
 - `cutoff`: the desired truncation error at each link.
 - `maxdim`: the maximum link dimension.
 """
 function (::Type{MPST})(
-  A::ITensor, sites; leftinds=nothing, orthocenter::Integer=length(sites), kwargs...
+  A::ITensor,
+  sites;
+  leftinds=nothing,
+  orthocenter::Integer=length(sites),
+  tags::Vector{TagSet}=[defaultlinktags(i) for i in 1:(length(sites) - 1)],
```

Comment:
```suggestion
  tags=[defaultlinktags(i) for i in 1:(length(sites) - 1)],
```
Tags can also be input as strings so it is better to not restrict the type.